std_msgs/Header header

PlaneData[] x_planes
PlaneData[] y_planes
